src/msix/common/AppxManifestObject.cpp

```diff
@@ -383,4 +385,88 @@ namespace MSIX {
         *documentElement = m_dom->GetDocument().As<IMsixElement>().Detach();
         return static_cast<HRESULT>(Error::OK);
     } CATCH_RETURN();
+
+    // Helper to get capabilities from the manifest
+    std::vector<std::string> AppxManifestObject::GetCapabilities(APPX_CAPABILITY_CLASS_TYPE capabilityClass)
+    {
+        // Windows code returns E_INVALIDARG for APPX_CAPABILITY_CLASS_DEFAULT.

```

Comment:
>// Windows code returns E_INVALIDARG for APPX_CAPABILITY_CLASS_DEFAULT. [](start = 8, length = 71)

What the what?

Reply:
lol? I'm going to treat APPX_CAPABILITY_CLASS_DEFAULT as APPX_CAPABILITY_CLASS_GENERAL

---
In reply to: [305428239](https://github.com/Microsoft/msix-packaging/pull/179#discussion_r305428239) [](ancestors = 305428239)